Sources/User.php

```diff
@@ -5271,6 +5271,21 @@ protected function setLanguage(): void
 		elseif (!empty($_SESSION['language']) && isset($languages[strtr($_SESSION['language'], './\\:', '____')])) {
 			$this->language = strtr($_SESSION['language'], './\\:', '____');
 		}
+		// Can we locate it in the accept language?
+		elseif (!empty($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
```

Comment:
```suggestion
		elseif (!empty($_SERVER['HTTP_ACCEPT_LANGUAGE']) && class_exists('\Locale')) {
```

The `\Locale` class is provided by the intl extension, but many hosts out there don't have the intl extension installed. Since trying to guess the right language based on `$_SERVER['HTTP_ACCEPT_LANGUAGE']` is a pleasant addition rather than a mission critical feature, I say we just skip the attempt if the class is unavailable.

Reply:
Makes sense.  It was a nice addition since we can now properly sniff the browser data to change the language.